public/scripts/calculator.js

```diff
@@ -0,0 +1,34 @@
+baseUrl ="http://localhost:8080/api/";
+
+// revisar fibonacci hay problemas
+$("#fib_button").click(function(){
+  fibUrl  = baseUrl + "fibonacci/" + $("#fib_number").val();;
+  console.log(fibUrl);
+  $.get(fibUrl, function(data, status){
+      if (data == "") {
+        $("#fib_result").text("0");
+      } else {
+        $("#fib_result").html(`${data}`);
```

Comment:
Isn't the `data` already a String?

```suggestion
        $("#fib_result").html(data);
```

I may be wrong. Applies throughout.

Reply:
Hi! When I use that jquery template tag .html(`${data}`) is showed in the form 1,1,2,3... but if use html(data) this is shown as 1123.

[Here](https://api.jquery.com/jquery.get/) says that: 

> The success callback function is passed the returned data, which will be an XML root element, text string, JavaScript file, or JSON object, depending on the MIME type of the response. It is also passed the text status of the response.

By the moment I let it as was because I think is clearer.